pkg/reconciler/v1alpha1/autoscaling/kpa/kpa_test.go

```diff
@@ -89,6 +89,8 @@ func newDynamicConfig(t *testing.T) *autoscaler.DynamicConfig {
 // TODO(#3591): Convert KPA tests to table tests.
 
 func TestMetricsSvcIsReconciled(t *testing.T) {
+	ClearAllLoggers()
```

Comment:
Would it be better to do defer ClearAllLoggers? We see, to do that pattern elsewhere and I think its nicer for tests being responsible to cleanup themselves

Reply:
Ehr, you're right! Will fix.

Reply:
Done!